ext/nnstreamer/tensor_decoder/meson.build

```diff
@@ -114,3 +114,27 @@ static_library('nnstreamer_decoder_image_segment',
   install: true,
   install_dir: nnstreamer_libdir
 )
+
```

Comment:
Make flatbuf optional. (enable flatbuf only if the corresponding meson option is enabled)